inst/scripts/make-grieneisen-ts-data.R

```diff
@@ -0,0 +1,111 @@
+library(SummarizedExperiment)
+library(TreeSummarizedExperiment)
+library(dplyr)
+library(phyloseq)
+
+#Sample phenotype data
+samples <- read.csv("git_metadata_and_community_phenotypes.csv")
+rownames(samples) <- samples$sample
+samples$sample <- NULL
+
+#Abundance table
+counts <- readRDS("git_ASV_table.RDS")
+
+#sequence information
+ASV_Sequence <- counts$asv_sequence
+ref_seq <- data.frame(ASV_Sequence)
+rownames(ref_seq) <- counts$asv_id
+
+#Taxonomic mapping table
+tax_1 <- select(counts, domain, phylum, class, order, family, genus, asv_id)
+rownames(tax_1) <- counts$asv_id
+
+# In order to make sure that the samples match between the abundance and phenodata tables
+counts_trimmed <- counts[, rownames(samples)]
+
+#Phylogenetic tree
+pseq <- readRDS("asv_10prev_phyloseq_for_lahti.RDS")
+tree <- phy_tree(pseq)
+
+#converting csv file into rds file
+saveRDS(samples, "samples.rds")

```

Comment:
You are saving all these data components as rds in the end of this script, no need to do intermediate saving. Remove line 31 and other similar saveRDS files, except the ones in the end of this script.